index.html

```diff
@@ -0,0 +1,349 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Anthill Inside</title>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+    <title>Deep Learning Conf – India’s first</title>
```

Comment:
Title is incorrect; Anthill Inside – India’s leading conference on Deep Learning and Artificial Intelligence